Tasks/Shurko/Task3/Task3.py

```diff
@@ -0,0 +1,59 @@
+while True:
+    s_long = input ("Enter string length from 35 to 50: ")
```

Comment:
wtf with the spaces between a function name and parentheses??? please learn to not add those, it breaks readability of the code